app/modules/flap/models_flappybird.py

```diff
@@ -0,0 +1,22 @@
+from datetime import datetime
+
+from sqlalchemy import ForeignKey, Integer, String
+from sqlalchemy.orm import Mapped, mapped_column, relationship
+
+from app.core.models_core import CoreUser
+from app.types.sqlalchemy import Base, TZDateTime
+
+
+class FlappyBirdScore(Base):
+    __tablename__ = "flappy-bird_score"
+
+    id: Mapped[str] = mapped_column(String, primary_key=True, index=True)
```

Comment:
You can use the new [`PrimaryKey`](https://github.com/aeecleclair/Hyperion/blob/5256b03a53e397dc710fdadf99a347ac31c4c92f/app/types/sqlalchemy.py#L36)
```suggestion
    id: Mapped[PrimaryKey]
```

Reply:
The PrimeryKey do not work because it is not a string